flink-scala/src/test/scala/org/apache/flink/api/operator/MaxByOperatorTest.scala

```diff
@@ -0,0 +1,176 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.api.operator
+
+import org.apache.flink.api.common.InvalidProgramException
+import org.apache.flink.api.scala._
+import org.junit.Test
+import org.junit.Assert
+
+class MaxByOperatorTest {
+  private val emptyTupleData = List[scala.Tuple5[Int, Long, String, Long, Int]]()
```

Comment:
Should this be `List[(Int, Long, String, Long, Int)]` or `List(Int, Long, String, Long, Int)`. Because if is the former 
`val collection = env.fromCollection(emptyTupleData)` throws compilation error
and if it is the latter the build fails
`[ERROR] /home/travis/build/apache/flink/flink-scala/src/test/scala/org/apache/flink/api/operator/MinByOperatorTest.scala:27: error: object java.lang.String is not a value
[ERROR]   private val emptyTupleData = List(Int, Long, String, Long, Int)`
This way 'List[scala.Tuple5[Int, Long, String, Long, Int]]()' was not throwing any exception.